File: solutions/python/darts/1/darts.py
from math import sqrt
def score(x, y):
    radius = sqrt(x**2 + y**2)
    if radius > 10:
        return 0
    if 5 < radius <= 10:
        return 1
    if 1 < radius <= 5:
        return 5
    if radius <= 1:
        return 10

File: solutions/cpp/darts/1/darts.cpp
#include "darts.h"

namespace darts {
int score (double x, double y){
    const double distance_squared = (x*x) + (y*y);
    if (distance_squared <= 1.0)   return 10;
    if (distance_squared <= 25.0)  return 5;
    if (distance_squared <= 100.0) return 1;
    else                           return 0;
}
}  // namespace darts